docs/content/topics/scripting/guide.md

```diff
@@ -9,1401 +9,867 @@ aliases:
-Creating secret do-nothing-1504055331341-master
-Creating Job Cache PVC github-com-brigadecore-empty-testbed-do-nothing
-Creating pod do-nothing-1504055331341-master
-Timeout set at 900000
-default/do-nothing-1504055331341-master phase Pending
-default/do-nothing-1504055331341-master phase Pending
-default/do-nothing-1504055331341-master phase Succeeded
-Destroying PVC named brigade-worker-01brx7v6wsg31k81x0h4pznv47-master
-Done in 5.12s.
-```
-
-That's quite a lot of new output for a program that "does nothing". The important part is
-this: `Creating pod do-nothing-1504055331341-master`. That tells us that it has taken our
-Job and packaged it up as what Kubernetes calls a _pod_. That means it has been scheduled
-for execution.
+$ b event create --project first-job --follow
 
-For a few lines we will see messages that let us know that our job isn't running yet, but
-is in state `Pending`. Then it will run, and if it runs to a successful completion, it
-will be marked as `Succeeded`.
+Created event "aa8fff14-0b8d-4903-9109-ccadc1d9d3fe".
 
-After that, our job is cleaned up, and so is the build.
-
-> Tip: You can see the actual output of each Job through the Brigade user interface, or using
-> the Kubernetes `kubectl` client: `kubectl logs do-nothing-1504055331341-master`
+Waiting for event's worker to be RUNNING...
+2021-09-22T18:41:01.787Z INFO: brigade-worker version: 927850b-dirty
+2021-09-22T18:41:02.130Z [job: do-nothing] INFO: Creating job do-nothing
+```
 
-Basically, our simple build just created an empty Alpine Linux pod which had nothing to do
+Basically, our simple build just created an empty Debian Linux pod which had nothing to do
 and so exited immediately.
 
-### Adding Tasks to Jobs
+### Adding Commands to Jobs
 
-To make our Job do more, we can add tasks to it. A task is an _individual step to be run
-inside of the Job's container_.
+To make our Job do more, we can add a _command_ to it. A command can then be
+paired with a list of _arguments_. The command and arguments array is added to
+the job's _primaryContainer_, which is the container running the image supplied
+to the `Job` constructor, e.g. `debian` in this example. (Every job will
+have one `primaryContainer` and zero or more `sidecarContainers`. We'll look
+at an example using `sidecarContainers` later on.)
 
 ```javascript
-const { events, Job } = require("brigadier")
+const { events, Job } = require("@brigadecore/brigadier");
 
-events.on("exec", () => {
-  var job = new Job("do-nothing", "alpine:3.4")
-  job.tasks = [
-    "echo Hello",
-    "echo World"
-  ]
+events.on("brigade.sh/cli", "exec", async event => {
+  let job = new Job("my-first-job", "debian", event);
+  job.primaryContainer.command = ["echo"];
+  job.primaryContainer.arguments = ["My first job!"];
+  await job.run();
+});
 
-  job.run()
-})
+events.process();
 ```
-[brigade-06.js](../../examples/brigade-06.js)
+[03-first-job](https://github.com/brigadecore/brigade/tree/v2/examples/03-first-job)
```

Comment:
See [here](https://github.com/brigadecore/brigade/pull/1629/files#r718631954). I think we used this example twice.